packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -19,31 +19,47 @@ import checkDuplicatedNodes from "babel-check-duplicated-nodes";
 
 import diff from "jest-diff";
 
-const moduleCache = {};
-const testContext = vm.createContext({
-  ...helpers,
-  process: process,
-  transform: babel.transform,
-  setTimeout: setTimeout,
-  setImmediate: setImmediate,
-  expect,
-});
-testContext.global = testContext;
+const contextModuleCache = new WeakMap();
```

Comment:
Comparing the CI build time between [this PR](https://travis-ci.com/github/babel/babel/builds/163788731) and [master](https://travis-ci.com/github/babel/babel/builds/163835216), the unit test now costs 30% longer time. My gut is it comes from the cost of creating every single context and evaluating `@babel/polyfill` and `babel-helpers-in-memory`.

Instead of running common modules for every single `exec` test, can we leverage the [`cacheData`](https://nodejs.org/api/vm.html#vm_vm_runincontext_code_contextifiedobject_options) option? We can generate code cache after running common modules, and supply it to every single `exec` test context.